docs/products/opensearch/howto/opensearch_alerting.rst

```diff
@@ -0,0 +1,120 @@
+-------------------
```

Comment:
The upper line is not needed. Let's remove it to be consistent with other articles